### ISSUE TITLE
Unmount issue with AxisTitle

### ISSUE THREAD

**Issue by SumBeam** [whawker/react-jsx-highcharts#24]
Hi, another unmount issue :(

I only have a minified stacktrace, but the code relates to this commit: https://github.com/whawker/react-jsx-highcharts/commit/78ceaabe76a212f1f066006fc5c531aed4bd5019

This was working in the 1.3.3 release.

```
// AxisTitle.js
componentWillUnmount () {
    if (this.props.getAxis()) { // getAxis throws the error
      this.updateAxisTitle({
        text: null
      });
    }
  }
```
```
highstock.js:formatted:1 Uncaught TypeError: Array.prototype.find called on null or undefined
    at find (native)
    at a.find (http://localhost:8080/bundle.js:43982:452)
    at a.Chart.get (http://localhost:8080/bundle.js:44208:343)
    at Object.v [as getAxis] (http://localhost:8080/bundle.js:70719:6862)
    at t.value (http://localhost:8080/bundle.js:70719:26192)
    at http://localhost:8080/bundle.js:63416:25
    at measureLifeCyclePerf (http://localhost:8080/bundle.js:63083:12)
    at ReactCompositeComponentWrapper.unmountComponent (http://localhost:8080/bundle.js:63415:11)
    at Object.unmountComponent (http://localhost:8080/bundle.js:5469:22)
    at ReactCompositeComponentWrapper.unmountComponent (http://localhost:8080/bundle.js:63425:23)
```

And also this one:

```
// Series.js
componentWillUnmount () {
    if (this.props.getSeries()) { // getSeries will throw the error below
      this.props.remove(); // Series may have already been removed, i.e. when Axis unmounted
    }
  }
```

```
Uncaught TypeError: Array.prototype.find called on null or undefined
    at find (native)
    at a.find (http://localhost:8080/bundle.js:43969:452)
    at a.Chart.get (http://localhost:8080/bundle.js:44195:343)
    at Object.p [as getSeries] (http://localhost:8080/bundle.js:70706:10589)
    at t.value (http://localhost:8080/bundle.js:70706:47999)
    at http://localhost:8080/bundle.js:63403:25
    at measureLifeCyclePerf (http://localhost:8080/bundle.js:63070:12)
    at ReactCompositeComponentWrapper.unmountComponent (http://localhost:8080/bundle.js:63402:11)
    at Object.unmountComponent (http://localhost:8080/bundle.js:5469:22)
    at ReactCompositeComponentWrapper.unmountComponent (http://localhost:8080/bundle.js:63412:23)
```

**Comment by SumBeam** [comment]
I have a workaround to disable XAxis.Title and YAxis.Title to fix the AxisTitle unmount issue.

I can also disable the series, but that's the whole point of highcharts ;)

```
<HighchartsStockChart>
  <Chart zoomType='x' />
  <Legend layout='horizontal' />

  <RangeSelector selected={5}>
    <RangeSelector.Button count={1} type='minute'>1m</RangeSelector.Button>
    <RangeSelector.Button count={1} type='hour'>1h</RangeSelector.Button>
    <RangeSelector.Button count={1} type='day'>1d</RangeSelector.Button>
    <RangeSelector.Button count={2} type='week'>2w</RangeSelector.Button>
    <RangeSelector.Button type='all'>All</RangeSelector.Button>
  </RangeSelector>

  <Tooltip />

  <XAxis id='time'>
    {/* <XAxis.Title>Time</XAxis.Title> */}
  </XAxis>

  <YAxis id='temperature'>
    {/* <YAxis.Title>Temperature (°C)</YAxis.Title> */}
    {
      _.map(series, (item, index) => {
        return <LineSeries key={index} id={index + 'key'} name={item.key} showInNavigator={item.showInNavigator === true} data={item.data} />
      })
    }
  </YAxis>
  <Scrollbar enabled={false} liveRedraw={false} />
  <Navigator />
</HighchartsStockChart>
```

**Comment by whawker** [comment]
Just published `react-jsx-highcharts@1.3.4-pre.2` and `react-jsx-highstock@1.3.4-pre.2` which should fix this issue.

Mind confirming and I'll publish a stable version?

**Comment by SumBeam** [comment]
Problem solved, works for me :) Thanks!
